package.json

```diff
@@ -16,6 +16,7 @@
     "test": "cross-env NODE_ENV=production node ./bin/test.js",
     "assert-code": "npm run eslint && npm run test",
     "validate-links": "node ./bin/remarkValidateLinks.js",
+    "travis-jobs": "npm run assert-code && npm audit --audit-level=low && npm run validate-links",
```

Comment:
Maybe neutralize the target name ? (name it `ci-jobs`, or  something similar, just for the day when we might move away from Travis-ci)

Reply:
One last request, could the developer readme be updated to match the change?